pom.xml

```diff
@@ -0,0 +1,139 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.microsoft.sqlserver.jdbc.spark</groupId>
+    <artifactId>spark-mssql-connector_2.11</artifactId>
+    <packaging>jar</packaging>
+    <description>spark-mssql-connector</description>
+    <version>1.0.0</version>
+    <name>spark-mssql-connector</name>
```

Comment:
@luxu1-ms If this is the name that the project will be listed by, please use this official name: Apache Spark Connector for SQL Server and Azure SQL